builtin/providers/rancher/resource_rancher_environment.go

```diff
@@ -203,6 +261,39 @@ func setOrchestrationFields(orchestration string, data map[string]interface{}) {
 	data[orch] = true
 }
 
+func normalizeMembers(in []rancherClient.ProjectMember) (out []interface{}) {
+	for _, m := range in {
+		mm := map[string]string{
+			"external_id_type": m.ExternalIdType,
+			"external_id":      m.ExternalId,
+			"role":             m.Role,
+		}
+		out = append(out, mm)
+	}
+	return
+}
+
+func makeProjectMembers(in []interface{}) (out []interface{}) {
+	for _, m := range in {
+		mMap := m.(map[string]interface{})
+		mm := rancherClient.ProjectMember{
+			ExternalIdType: mMap["external_id_type"].(string),
+			ExternalId:     mMap["external_id"].(string),
+			Role:           mMap["role"].(string),
+		}
+		out = append(out, mm)
+	}
+	return
+}
+
+func memberHash(member interface{}) int {
```

Comment:
Actually, since members are resources and not hashes, it seems I need a Set function.